app/controllers/dashboard_controller.rb

```diff
@@ -2,11 +2,15 @@
 
 class DashboardController < ApplicationController
   def index
-    @reasons = Reason.all.joins(:posts)
-                     .group('reasons.id')
-                     .select('reasons.*, count(\'posts.*\') as post_count')
-                     .order('inactive ASC, post_count DESC')
-                     .to_a
+    @inactive_reasons, @active_reasons = [true, false].map do |inactive|
+      Reason.all.joins(:posts)
+            .where("reasons.inactive = #{inactive}")
+            .group('reasons.id')
+            .select('reasons.*, count(\'posts.*\') as post_count')
+            .order('post_count DESC')
+            .to_a
+    end
+    @reasons = Reason.all
```

Comment:
Is this ever used in the view?

Reply:
Yes, `@reasons` is used in the ["&lt;n> filters have caught &lt;m> posts"](https://github.com/serixscorpio/metasmoke/blob/4f5c72f91470c39cd5a2e16fc7b164cdb1df6008/app/views/dashboard/index.html.erb#L2-L3) paragraph above the first reasons table.   I wasn't sure if the number of filters displayed here should count all reasons or just active reasons, so I left the behavior unchanged.  Some optimization is possible such as using `@active_reasons.count + @inactive_reasons.count` instead of `@reasons.count`.